models/aavegotchi/polygon/aavegotchi_polygon_schema.yml

```diff
@@ -0,0 +1,213 @@
+version: 2
+
+models:
+  - name: aavegotchi_polygon_events
```

Comment:
where is the unique test here? for incremental models, we'd like to see an unique test run on the unique key columns of the model. this can be either at column level (if single column is unique) or with dbt unique combo of columns (found in many other model schemas)

Reply:
Added to the unique_trade_id field.